src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
 export interface Figure {
-
+  shape: string,
+  color: string,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape;
 
-}
+  public sides;
+
+  constructor(public color: 'red' | 'green' | 'blue', ...args: number[]) {
```

Comment:
use expected values instead of args, you know you want to only get 3 sides of the triangle, no more or no less, you shouldn't use ...args in this case